core/src/definitions-internal.ts

```diff
@@ -165,9 +164,15 @@ export interface StoredCallback {
   reject?: (...args: any[]) => any;
 }
 
+export interface CapacitorCustomPlatformInstance {
+  name: string;
+  useWebFallbackPlugins?: boolean;
```

Comment:
Would platforms have any reason to set this to false?
I see you set it to true in electron v4, and I think all platforms should fallback to the web plugin when there is no native implementation.

Reply:
This isnt needed, I just forgot to take this out after my testing. Sorry